cpp/src/copying/contiguous_split.cu

```diff
@@ -80,16 +94,22 @@ struct column_buf_size_functor {
 
    template <typename T, std::enable_if_t<is_fixed_width<T>()>* = nullptr>
    std::pair<size_t, size_t> operator()(std::pair<size_t, size_t> sizes, column_view const& c)
-   {
-      size_t data_size = sizes.first + (cudf::util::div_rounding_up_safe(c.size() * sizeof(T), split_align) * split_align);
+   {      
+      size_t data_size = sizes.first + cudf::util::round_up_safe(c.size() * sizeof(T), split_align);  
       size_t validity_size = sizes.second + (c.nullable() ? cudf::bitmask_allocation_size_bytes(c.size(), split_align) : 0);
```

Comment:
You don't even need the second argument to `cudf::bitmask_allocation_size_bytes()`, since it defaults to 64.